kie-dmn-feel/src/main/java/org/kie/dmn/feel/lang/ast/BaseNode.java

```diff
@@ -119,9 +124,17 @@ public String toString() {
     public Type getResultType() {
         return BuiltInType.UNKNOWN;
     }
+    
+    protected Supplier<FEELEvent> astEvent(Severity severity, String message) {
```

Comment:
Helper methods to keep it short when creating `FEELEvent`s for AST node problems